modules/@apostrophecms/i18n/i18n/en.json

```diff
@@ -45,6 +45,11 @@
   "assetWebpackCacheCleared": "Build cache cleared.",
   "assetWebpackConfigWarning": "⚠️ In the module {{ module }}, your webpack config is incorrect. It must be an object and should contain only the properties: {{ properties }}.",
   "at": "at",
+  "automaticTranslationCheckbox": "Translate text content",
+  "automaticTranslationSettings": "Automatic translation settings",
+  "automaticTranslationSourceErrMsg": "The current locale <strong>{{ source }}</strong> is not suitable for translation. Translation cannot happen.",
```

Comment:
"Translation will be skipped" probably? Not sure though, native English speakers have to advise.

Reply:
Right, I updated that.